data/base/sles/sle15/config.yaml

```diff
@@ -0,0 +1,34 @@
+config:
+  SLES:
+    sysconfig:
```

Comment:
This information will be used to create config.sh correct ? As config.sh is a custom script that can contain anything do we have an idea how raw scripting should be represented in yaml. I'm asking this because raw data in yaml and also json has some pitfalls e.g on line breaks

So how would we represent this script part as example:

```
pushd /etc/products.d
ln -sf SLES_SAP.prod baseproduct
popd
```

Reply:
The `config` section isn't used yet, this is basically a left-over from the initial example. I'll work on it next.